addons/custom-block-shape/userscript.js

```diff
@@ -0,0 +1,246 @@
+export default async function ({ addon, global, console }) {
+  var multiplier = addon.settings.get("paddingSize") / 100;
+  var BlocklyInstance = await addon.tab.traps.getBlockly();
+
+  (function (Blockly) {
+    const BlockSvg = BlocklyInstance.BlockSvg;
+    var vm = addon.tab.traps.vm;
+
+    const { GRID_UNIT } = BlockSvg;
+    var cornerSize = 1;
+    var notchSize = 1;
```

Comment:
ditto

Reply:
The 1s are also leftovers and can be removed.